.design-system-version

```diff
@@ -1 +1 @@
-45.2.0
+46.1.3
```

Comment:
For some reason, this isn't available via the AWS CDN, which means this can't go to preprod until it is fixed. Try to change the local CDN to AWS one, all the assets will be broken. Will progress via Slack.

Reply:
Will investigate separately why 46.1.3 is now available.